tests/test_fd.py

```diff
@@ -28,6 +29,19 @@ def test_FD(self):
         # Those two clock correction difference are causing the trouble.
         assert np.all(resDiff< 5e-6) , \
             "PINT and tempo Residual difference is too big. "
+    def test_inf_freq(self):
+        test_toas = copy.deepcopy(self.toas)
+        test_toas.table['freq'][0:5] = np.nan
```

Comment:
I think this should be np.inf * u.MHz (i.e. it should not be np.nan and it should have units)